macos/FluentUI/Resources/FluentUI-macos.xcassets/ButtonColors/secondOuterDropShadow.colorset/Contents.json

```diff
@@ -0,0 +1,38 @@
+{
+  "colors" : [
+    {
+      "color" : {
+        "color-space" : "srgb",
+        "components" : {
+          "alpha" : "0.150",
+          "blue" : "0x00",
+          "green" : "0x00",
+          "red" : "0x00"
+        }
+      },
+      "idiom" : "universal"
+    },
+    {
+      "appearances" : [
+        {
+          "appearance" : "luminosity",
+          "value" : "dark"
+        }
+      ],
+      "color" : {
+        "color-space" : "srgb",
+        "components" : {
+          "alpha" : "0.250",
```

Comment:
Alphas are still decimal, not hex? I think designers specify these using hex so there's still a lossy conversion?

Reply:
Erm, isn't alpha a %? How would you specify that in Hex? Also all the pre-existing ColorSets in ButtonColors have a decimal format alpha

Reply:
An 8 digit hex string. An example:
https://github.com/microsoft/fluentui-design-tokens/blob/main/src/black.json#L14

If Xcode doesn't let you / your designs are in decimal, then that's fine.

Reply:
the example you linked in a color and not alpha value? I can't find out how to express alpha in hex through Xcode, and don't see examples of this used in Contents.json so just going with what we have for now

Reply:
For `"#0000001f"`, the first 6 digits are the color, and the last 2 (1f) is the alpha. But no worries, if Xcode can't do it then that's fine :)